dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -1050,6 +1107,13 @@ public ResponseEntity<RepresentationModel<?>> delete(HttpServletRequest request,
         return deleteInternal(apiCategory, model, uuid);
     }
 
+    @RequestMapping(method = RequestMethod.DELETE, value = REGEX_REQUESTMAPPING_IDENTIFIER_AS_STRING_VERSION_STRONG)
+    public ResponseEntity<RepresentationModel<?>> delete(HttpServletRequest request, @PathVariable String apiCategory,
```

Comment:
New method is missing JavaDocs.  Again, we should add tests for this new method to `RestResourceControllerIT`

Reply:
methods `RestResourceController.delete(..., String);` | `RestResourceController.upload(..., String);` | `RestResourceController.patch(..., String);` have been added because they are used into `QAEventRestRepository`. Goal is the reflection is able to manage these calls and handle them properly to `QAEventRestRepository`. Consider **`QAEventRestRepositoryIT`** tests these methods already.